### PR TITLE
Automated test for multi host credentialing

### PR DESCRIPTION
**Pull request by nachandr**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7fbabfeb8427a2eb3a2a761899b5a6a56d3af5d9:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_host.py:33:19: F821 undefined name 'validate_credentials'
```
:trollface:
*CFME QE Bot*

File: cfme/tests/infrastructure/test_host.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    quads = Quadicon.all("host", this_page=True)
+    for quad in quads:
+            sel.check(quad.checkbox())
+    tb.select("Configuration", "Edit Selected items")
+
+    cfme_host = random.choice(provider.get_yaml_data()["hosts"])
+
+    fill(credential_form, {'default_principal': 'root',
+        'default_secret': 'dog8code',
```
Comment (mfalesni) [inline cfme/tests/infrastructure/test_host.py]: I don't think that sharing this is a good idea. I believe the host creds can be scavenged from the YAMLs.

Please remove the password ASAP.

Comment (seandst) [inline cfme/tests/infrastructure/test_host.py]: In fact, it's a very bad idea. We put a fair amount of effort to keep passwords centrally configured and out of publicly shared revision control; please don't subvert those systems.

Comment (nachandr) [inline cfme/tests/infrastructure/test_host.py]: Oh yes,that is on my to do list.I was trying to get something else to work,so forgot to remove the host creds.

**Comment by dajoRH** [comment]
Lint report for commit b96b1ba818b7776e5b50d39d926d7db544795b6e:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_host.py:32:19: F821 undefined name 'validate_credentials'
```
:trollface:
*CFME QE Bot*

**Comment by mfalesni** [comment]
@seandst @nachandr I allowed myself to delete the three comments in order to remove links to the stale commit with the leak. I hope it did not offend anyone :)

**Comment by dajoRH** [comment]
Lint report for commit d336a76c95e1d1aa72a171424f99b2c10ce6d829:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 89f5eae8d07b0fa728f2b1b3d8a7171c939927c1:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6fe435787232cf4f13c684f08509a7e1a8f59fd6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 02d51063a1dce85af51ae2b79d4cec67a27accce:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c8cbadf962718877f7abd97406cdc043ced4004b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5f737209958cc685d585587f3b903c838b99cefc:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a29a028847886a918e8f2e747c903310fd7e1397:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9ce0ce4c8537047c4b27e2d65a1b46d25d7204ed:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit adb89e54193ac3b1f42365ee8f13714fdf5bb0c6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/infrastructure/test_host.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+from cfme.infrastructure.host import credential_form
+from cfme.infrastructure.provider import details_page
+from cfme.web_ui import Quadicon, fill
+from utils import testgen
+from utils import version
+
+
+def config_option():
+    version.pick({version.LOWEST: 'Edit Selected Hosts', '5.4': 'Edit Selected items'})
```
Comment (seandst) [inline cfme/tests/infrastructure/test_host.py]: return

**Comment by dajoRH** [comment]
Lint report for commit 339792b713016769ac15de6ec899809c46c65868:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**